EIPS/eip-1949.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: 1949
+title: Delayed Distributed Minting
+author: Johann Barbie (@johannbarbie), Pinkiebell (@pinkiebell)
+discussions-to: https://ethresear.ch/t/a-distributed-breeding-function/5264
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-04-18
+requires: 721, 1948
+---
+
+## Simple Summary
+
+```
+Two CryptoKitties take a honeymoon on a sidechain 🐈🐈🏖. The blazing fast transactions heat their blood 🔥, one thing leads to another ❤ and a rare baby-kitten is born 💥🐯. On the way home to mainnet the family gets stopped at the bridge contract, “👮‍♂️ tokenIDs please!”. Oh shoot, the baby-kitten doesn’t have one on the root chain 🤷‍♀️.
+```
+
+The minting function of NFT contracts is centralized to a single chain. This proposal allows to mint tokens across multiple chains.
+
+## Abstract
+
+This ERC introduced a design for NFT tokens to be mintable across multiple chains. A minimal requirement for distributed minting are collision-free token Ids. The collision-freeness is achieved by introducing two types of tokens, queens and workers. The Ids of the queens are generated by hashing the token contract address with an increasing `queenCounter` in the contract. The Ids of the workers are generated by hashing the queenIds with an increasing counter that the queen tokens carry in themselves using the [data token specification](https://github.com/ethereum/EIPs/pull/1948).
+
+## Motivation
+
+While fungible tokes have a natural performance gain when using sidechains, NFTs need to be minted on the mainnet and deposited one by one. With the distributed breeding function described here, NFTs can be minted on sidechains and only cause mainnet costs on exit.
+
+## Specification
+
+An extension of [ERC721](https://eips.ethereum.org/EIPS/eip-721) interface with the following function is suggested:
+
+```
+pragma solidity 0.5.2;
+
+/**
+ * @title IERC1949
+ * @dev Interface for delayed breeding
+ */
+contract IERC1949 {
+
+  function breed(uint256 _workerId, bytes32 _workerData) public;
```

Comment:
The ERC talks about minting, but the wording here is breeding. Which one is correct?